src/run.ts

```diff
@@ -108,6 +260,7 @@ export async function runPublish({
         createRelease(octokit, {
           pkg,
           tagName: `${pkg.packageJson.name}@${pkg.packageJson.version}`,
+          comment: false
```

Comment:
why this is hardcoded to false? 🤔 shouldn't we use a configured value here?

Reply:
This is hardcoded to false because we want to comment on whole repository release rather than on each package release.

Reply:
Hm, there is no such thing as "whole repository release" if I recall correctly. The other `createRelease` is for single-package repositories but in monorepos we are just looping through all released packages so there is 1 to 1 relation between released packages and created releases (no extra repository-wide release)

Reply:
Oh right I misunderstood what @mitchellhamilton meant here https://github.com/changesets/bot/pull/25#pullrequestreview-618021405

I'll use the `comment` value from the input property.